rebar.config

```diff
@@ -6,7 +6,7 @@
 
 {project_plugins, [rebar3_proper,
                    rebar3_hex,
-                   {rebar3_edoc_extensions, "1.0.0"}]}.
+                   rebar3_edoc_extensions]}.
```

Comment:
Could you please remove this unrelated change?

Reply:
Done. Do I need move those changes in separate commit or open new PR for it? Or for now this is not necessary at all?